idt_config/windows/userdata.json

```diff
@@ -0,0 +1,52 @@
+{
+  "sourcePath": "<path to project root, e.g. C:/Users/foobar/iot-reference-nxp-rt1060>",
+  "freeRTOSTestParamConfigPath": "{{testData.sourcePath}}/components/FreeRTOS-Libraries-Integration-Tests/config/test_param_config.h",
+  "freeRTOSTestExecutionConfigPath": "{{testData.sourcePath}}/components/FreeRTOS-Libraries-Integration-Tests/config/test_execution_config.h",
+  "freeRTOSVersion": "202012.04-LTS",
+  "buildTool": {
+    "name": "esp32c3-build",
+    "version": "1.0.0",
+    "command": [
+      "{{config.idtRootPath}}/configs/build.bat {{testData.sourcePath}}"
+    ]
+  },
+  "flashTool": {
+    "name": "esp32c3-flash",
+    "version": "1.0.0",
+    "command": [
+      "{{config.idtRootPath}}/configs/flash.bat {{testData.sourcePath}}"
+    ]
+  },
+  "testStartDelayms": 5000,
+  "echoServerCertificateConfiguration": {
+    "certificateGenerationMethod": "Automatic"
+  },
+  "echoServerConfiguration": {
+    "keyGenerationMethod": "EC",
+    "serverPort": 9000
+  },
+  "otaConfiguration": {
```

Comment:
This section has been updated in IDT, this configuration should be cleaned up based on the latest IDT version. You can sync with me for more information if you need help.

Reply:
otaConfiguration* to be more clear.

Reply:
Updated. Thanks for comment.